tests/yadism/test_tmc.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, tmc, xg=None):
         self.runner.configs = MockObj()
         self.runner.configs.M2target = 1.0
         self.runner.configs.TMC = tmc
-        self.runner.configs.interpolator = InterpolatorDispatcher(xg, 1, False, False)
+        self.runner.configs.interpolator = InterpolatorDispatcher(xg, 1, False)
```

Comment:
this is the `log=False` I meant [here](https://github.com/NNPDF/yadism/pull/157/files#r1013810070)